playbooks/roles/airship-configure-worker/tasks/subvolumes-setup.yml

```diff
@@ -22,12 +22,14 @@
     force: yes
 
 - name: Execute subvolume setup script
-  shell: /tmp/caasp_worker_node_set_subvolumes.sh
+  command: /tmp/caasp_worker_node_set_subvolumes.sh
   register: subvolume_cmd_out
   failed_when: subvolume_cmd_out.rc != 0
+  tags:
```

Comment:
Good to move to command. It would be possible to use script too, if the script doesn't need any templating. It would be more efficient too.